src/MainWindow.vala

```diff
@@ -40,6 +40,11 @@ public class AppCenter.MainWindow : Hdy.ApplicationWindow {
     private Gee.LinkedList<string> return_button_history;
     private Gtk.Label updates_badge;
     private Gtk.Revealer updates_badge_revealer;
+    private Gtk.Overlay overlay;
```

Comment:
It doesn't look like overlay is accessed outside of the construct block, so it can probably be scoped down to `var overlay` instead of declared here